res/skins/Deere/skin.xml

```diff
@@ -17,8 +17,28 @@
       <attribute persist="true" config_key="[Deere],show_parallel_waveforms">0</attribute>
       <attribute persist="true" config_key="[Microphone],show_microphone">0</attribute>
       <attribute persist="true" config_key="[PreviewDeck],show_previewdeck">0</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
```

Comment:
The show_library button was replaces with maximize_library in #940. After this commit, there are both buttons and they use the same icon, which is confusing.